### ISSUE TITLE
Content dismissal

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#110]
**URL:** https://spotlight.library.tamu.edu/spotlight/
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.4.13 Content on Hover or Focus (AA)
**Issues:**  Content dismissal
**Notes:**  New content can't be dismissed without moving the pointer

**Comment by qtamu** [comment]
This card was created as a mistake. This is not a violation.
